### DISCUSSION TITLE
Use Exemplars Preview

### DISCUSSION THREAD

**Discussion by gabswb** [open-telemetry/opentelemetry-cpp#1917] [Q&A]
Hi,
I would like to use exemplars but I have seen that it's still experimental in opentelemetry-cpp. Can we use the preview version ?
I have compiled opentelemetry-cpp with `DWITH_METRICS_EXEMPLAR_PREVIEW` and defined `ENABLE_METRICS_EXEMPLAR_PREVIEW`.
Should I do anything more ?

**Comment by lalitb** [comment]
Sorry for late reply. As you rightly mentioned, Exemplar is in preview, and mayn't be complete or stable. This will be revisited again sometime during next month based on priority and bandwidth of contributors.
